From: Katsuhiko Nishimra <ktns.87@gmail.com>
Date: Thu, 25 Jun 2015 15:01:07 +0900
Subject: Don't remove header files

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a7c090b..716dd6a 100644
--- a/Makefile
+++ b/Makefile
@@ -93,7 +93,7 @@ distclean::
 	    (cd $${dir} && $(MAKE) $(DODEPENDOPT) distclean) || exit 1; \
 	  done
 	-rm -rf autom4te.cache config.status config.log conf*.file conf*.dir *.dSYM depcheck* libtool \
-Makedirlist libint2.pc include/libint2
+Makedirlist libint2.pc
 
 targetclean::
 	for dir in $(SUBDIRS); \
